README.md

```diff
@@ -461,6 +461,7 @@ interface {
   firmwareFilename: string;
   firmwareFile: string; // use base64 encoding for the file
   firmwareFileFormat?: FileFormat;
+  target?: number;
```

Comment:
If we should always let target specify it, why not make it required?

Reply:
sorry I meant that the user should always be able to override it, but they don't have to.